cfme/infrastructure/resource_pool.py

```diff
@@ -83,7 +80,7 @@ def get_detail(self, *ident):
         Returns: A string representing the contents of the InfoBlock's value.
         """
         if not self._on_detail_page():
-            sel.force_navigate('infrastructure_resource_pool', context=self._get_context())
+            navigate_to(self, 'Details')
```

Comment:
You can remove the 
```if not self._on_detail_page()``` and dedent the ```navigate_to``` as this should always run and the ```am_i_here()``` will take care of the conditional